core/src/mindustry/game/MapObjectives.java

```diff
@@ -1285,11 +1287,16 @@ private void setUv(int i, double u, double v){
             if(i >= 0 && i < 4){
                 if(fetchedRegion == null) setTexture(textureName);
 
-                if(!Double.isNaN(u)) vertices[i * 6 + 3] = Mathf.map(Mathf.clamp((float)u), fetchedRegion.u, fetchedRegion.u2);
-                if(!Double.isNaN(v)) vertices[i * 6 + 4] = Mathf.map(1 - Mathf.clamp((float)v), fetchedRegion.v, fetchedRegion.v2);
+                if(!Double.isNaN(u)) {
```

Comment:
incorrect formatting in several places - space before {, redundant space after `if` above, missing space before `return` above